subworkflows/local/mapping_depth.nf

```diff
@@ -0,0 +1,59 @@
+
+/*
+ * Bowtie2 build and align steps for mapping
+ */
+
+include {BOWTIE2_ASSEMBLY_BUILD                          } from '../../modules/local/bowtie2_assembly_build'
+include {BOWTIE2_ASSEMBLY_ALIGN                          } from '../../modules/local/bowtie2_assembly_align'
+include {METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS            } from '../../modules/nf-core/metabat2/jgisummarizebamcontigdepths/main'
+
+workflow MAPPING_DEPTH {
+    take:
+    assemblies
+    reads
+
+    main:
+    ch_versions = Channel.empty()
+
+    // build index
+    BOWTIE2_ASSEMBLY_BUILD ( assemblies )
```

Comment:
What is the convention for the spacing around these calls? They're kinda all over the place

Reply:
Scattered across nf-core pipelines and modules (and I assume with subworkflows to come) is the non-consistent use of spacing, or extra spacing. Currently I assume they do this for readability but it actually makes things cumbersome down the line so I will get rid of extra spacing.